src/components/NoticeIcon/index.tsx

```diff
@@ -135,6 +133,8 @@ export default class NoticeIcon extends Component<NoticeIconProps> {
     }
   };
 
+  public static Tab: typeof NoticeList = NoticeList;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declaration of public static field not allowed after declaration of public instance method. Instead, this should come at the beginning of the class/interface.](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4526098)